Assets/HoloToolkit-Examples/Input/Scripts/XboxControllerHandlerTest.cs

```diff
@@ -30,13 +30,15 @@ protected virtual void Start()
             initialPosition = transform.position;
         }
 
+        //// Child classes can override the parent Start() class
         public override void OnSourceLost(SourceStateEventData eventData)
         {
             Debug.LogFormat("Joystick {0} with id: \"{1}\" Disconnected", GamePadName, eventData.SourceId);
             base.OnSourceLost(eventData);
             debugText.text = "No Controller Connected";
         }
 
+        //// Child classes can override the parent Start() class
```

Comment:
This comment doesn't seem to clarify anything. Why does it refer to the Start() function? Let's just remove this comment